### PR TITLE
Get feed with matched information

### PR DESCRIPTION
**Pull request by Es7evam**
usecases/offerCases.go  | Add GetMatchOffers -> returns the offers with matched bool (status)
    handlers/offerHandler.go | Add GetMatchedFeed -> Returns the feed with matched bool at offer
    handlers/router.go | Add GET calling GetMatchedFeed

    docs/API.md | Add docs of handler (GET at /offers/users/<userid>)
     -   GET /offers/users/<userid>?size=<int>
     -   GET /offers/users/<userid>?size=<int>&before=time

### CODE REVIEW

File: backend/usecases/offerCases.go [hunk 1]
```diff
@@ -90,3 +90,23 @@ func (oc *OfferController) InsertOfferMatch(userid int, offerid string) error {
 	}
 	return nil
 }
+
+// GetMatchOffers returns the time, size and the userid and it returns the offers with the information that is has matched or not
```
Comment (mbotarro) [inline backend/usecases/offerCases.go]: Is this comment correct?

File: backend/usecases/offerCases.go [hunk 2]
```diff
@@ -90,3 +90,23 @@ func (oc *OfferController) InsertOfferMatch(userid int, offerid string) error {
...
+func (oc *OfferController) GetMatchOffers(before time.Time, size, userid int) ([]models.MatchedOffer, error) {
+	offers, err := oc.offerDAL.GetLastOffers(before, size) // retorna offers
+	if err != nil {
+		return nil, err
+	}
+
+	// For each offer checks if it has matched and then updates the status
+	moffs := []models.MatchedOffer{}
+	for _, o := range offers {
+		status, _ := oc.offerDAL.GetOfferMatchStatus(userid, o.ID)
```
Comment (mbotarro) [inline backend/usecases/offerCases.go]: You might want to check for error!!

Comment (Es7evam) [reply backend/usecases/offerCases.go]: fixed at a783b99

File: backend/usecases/offerCases.go (hunk 1 above)
Comment (Es7evam) [reply backend/usecases/offerCases.go]: fixed at a783b99

**Pull request merged**